legate/tester/stages/util.py

```diff
@@ -25,10 +25,18 @@
 from ..logger import LOG
 from ..test_system import ProcessResult
 
-CUNUMERIC_TEST_ARG = "-cunumeric:test"
-
 UNPIN_ENV = {"REALM_SYNTHETIC_CORE_MAP": ""}
 
+CUNUMERIC_TEST_ENV = {"CUNUMERIC_TEST": "1"}
+
+# Raise min chunk sizes for deferred codepaths to force eager execution
+EAGER_ENV = {
+    "CUNUMERIC_MIN_CPU_CHUNK": "2000000000",
```

Comment:
Let's add `"CUNUMERIC_TEST": "0"` to this. The eager tests need to run with `CUNUMERIC_TEST` set to false, which is the default, but now it will be easier for users to unwittingly overwrite this setting by randomly having `CUNUMERIC_TEST` set in their environment, so better to make it explicit.

Reply:
[16598d5](https://github.com/nv-legate/legate.core/pull/572/commits/16598d58a9f1f8ff6ce28003e6a85dd7ef2d0660)